app/src/main/java/org/grapheneos/apps/client/App.kt

```diff
@@ -103,6 +104,10 @@ class App : Application() {
         }
 
         private fun getContext() = context.get()!!
+
+        @SuppressLint("StaticFieldLeak")
+        // not a memory leak: uses an application Context
+        lateinit var jobPsfsMgr: JobPsfsMgr
```

Comment:
IMHO avoid using laterinit if possible, everything else look good to me.

Reply:
"jobPsfsMgr" is guaranteed to be initialized by `App.onCreate()`

Reply:
yes, I know that. just saying you can use lazy initialization or something.